rpcs3/main.cpp

```diff
@@ -368,7 +368,6 @@ struct fatal_error_listener final : logs::listener
 // Arguments that force a headless application (need to be checked in create_application)
 constexpr auto arg_headless     = "headless";
 constexpr auto arg_decrypt      = "decrypt";
-constexpr auto arg_commit_db    = "get-commit-db";
```

Comment:
You can remove these includes now:

![image](https://github.com/user-attachments/assets/a10973cf-90a0-4f49-b1e8-464a5b9d0e69)